modules/apps/calendar/calendar-web-test/src/testFunctional/macros/CalendarPermissions.macro

```diff
@@ -0,0 +1,68 @@
+definition {
+
+	var siteName = "Liferay";
+
+	macro changePaginationPermissions {
+		Click(locator1 = "CalendarPermissions#PERMISSIONS_ITEMS_PER_PAGE_SELECT");
+
+		var key_itemsPerPage = "${itemsPerPage}";
+
+		MenuItem.click(menuItem = "${itemsPerPage}");
+
+		AssertTextEquals.assertPartialText(
+			locator1 = "Pagination#ITEMS_PER_PAGE_SELECT",
+			value1 = "${itemsPerPage} Entries");
+	}
+
+	macro cleanSearchResult {
+		Click(locator1 = "CalendarPermissions#PERMISSIONS_SEARCH_CLEAR");
+	}
+
+	macro gotoPermissions {
+		Portlet.gotoPortletOptions(
+			portletName = "Calendar",
+			portletOption = "Permissions");
+
+		SelectFrame(locator1 = "IFrame#MODAL_BODY");
+	}
+
+	macro resultBarIsnotDisplayed {
+		AssertElementNotPresent(locator1 = "CalendarPermissions#PERMISSIONS_RESULTS_BAR");
+	}
+
+	macro search {
```

Comment:
Please, change this macro name to searchRoleName, in order to make it easier to understand whats is being searched.

Reply:
Done!